android/src/miga/ticrypt/TiCryptModule.java

```diff
@@ -62,7 +61,7 @@ public TiCryptModule() {
         try {
             byte[]encodedKey = Base64.decode(keyString, 0);
             PKCS8EncodedKeySpec x509KeySpec = new PKCS8EncodedKeySpec(encodedKey);
-            KeyFactory keyFact = KeyFactory.getInstance("RSA", "BC");
+            KeyFactory keyFact = KeyFactory.getInstance("RSA");//fix:https://android-developers.googleblog.com/2018/03/cryptography-changes-in-android-p.html
```

Comment:
what about devices < Android P? Should there be a API version check here?

Reply:
it will keep working, the current method signature without arguments exists in previous version but after Android P it will be removed. I tested under an Huawai P10 with Android nougat.